docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,87 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |set certain contacts as favourites |access their information easily
+
+|`* * *` |user |select multiple contacts to delete |clean up address book efficiently
+
+|`* * *` |user |save more than one contact number per contact in my address book |account for multiple phone numbers
+
+|`* * *` |organised user |group contacts manually or automatically |access the information I want easily
+
+|`* * *` |user with busy schedules |see my important events on a calendar view |have an overview of my schedule for the month
+
+|`* * *` |organized user |share my contact details with my friends |save the contacts without manually keying in the details
+
+|`* * *` |impatient user |sort the information on the application |view the information in a ordered manner
+
+|`* * *` |user |add information to the application in a intuitive way |feel less frustrating
+
+|`* * *` |user |sync my account with other platforms |save the trouble of entering my personal details
+
+|`* * *` |user |access my data over multiple authorized devices |avoid keying my data repeatedly across different platforms
+
+|`* * *` |forgetful user |have my account logged in every time I open the app |do not have to remember one additional username and password
+
+|`* * *` |forgetful user |set reminders |do not forget what I need to do
+
+|`* * *` |user |export the list of people in my address book |store it on the computer
+
+|`* * *` |user |set events and send the event invites to any person in my contact list |sync my calendars with others' on our common events
+
+|`* * *` |user |attach personal notes to people in my contact list |take notes about the person for my own reference
+
+|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else
+seeing them by accident
+
+|`* *` |user |send mass messages to a group of contacts |plan for meetings, conferences or gatherings with ease
+
+|`* *` |user |drag and drop contacts in or out of different groups |not need to key in complex command lines
```

Comment:
Take note that one of the project constraints is to be primarily used in CLI.

Reply:
Okay.